rtc_auth_enclave/src/jwt.rs

```diff
@@ -0,0 +1,87 @@
+use std::string::{String, ToString};
+use std::time::{SystemTime, UNIX_EPOCH};
+
+use jsonwebtoken::{encode, EncodingKey, Header};
+use serde::{Deserialize, Serialize};
+use sgx_tstd::untrusted::time::SystemTimeEx;
+use uuid::Uuid;
+
+use crate::uuid_to_string;
+
+/// Claims body of the JWT token
+///
+/// Example output:
+/// ```
+/// {
+///     "iss": "registree_auth_enclave",
+///     "nbf": 1623762799,
+///     "iat": 1623762799,
+///     "jti": "b300fe149d144e05aa9a9600816b42ca",
+///     "exec_module_hash": "AAECAwQFBgcICQoLDA0ODxAREhMUFRYXGBkaGxwdHh8=",
+///     "dataset_uuid": "dd12012195c04ae8990ebd2512ae03ab"
+/// }
+/// ```
+#[derive(Debug, Serialize, Deserialize)]
+struct Claims {
```

Comment:
```suggestion
struct Claims {
    // Registered Claim Names: https://datatracker.ietf.org/doc/html/rfc7519#section-4.1
```

(Because I'm probably going to have to look up these acronyms every other time I look at this code. 😄️)